pkg/metadata/inventories_collector.go

```diff
@@ -48,7 +48,7 @@ func (c inventoriesCollector) Send(ctx context.Context, s *serializer.Serializer
 
 // Init initializes the inventory metadata collection
 func (c inventoriesCollector) Init() error {
-	return inventories.StartMetadataUpdatedGoroutine(c.sc, config.Datadog.GetDuration("inventories_min_interval")*time.Second)
+	return inventories.StartMetadataUpdatedGoroutine(c.sc, time.Duration(config.Datadog.GetInt("inventories_min_interval"))*time.Second)
```

Comment:
Do we get an acceptable error when a customer had a duration here that "worked" (bad value but it didn't crash) but now it might crash?

Reply:
Edit: Do we need/want some test around this?

Reply:
These are new, unpublished config fields, so I think we're OK in this case.